standards/genc/src/main/java/org/codice/countrycode/standards/genc/CodeUriSet.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.countrycode.standards.genc;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(namespace = "http://api.nsgreg.nga.mil/schema/genc/3.0/genc-cmn")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class CodeUriSet {
+
+  @XmlElement(
+    name = "codespaceURNBasedShort",
+    namespace = "http://api.nsgreg.nga.mil/schema/genc/3.0/genc-cmn"
```

Comment:
Can the namespace be pulled out into a variable?

Reply:
it could. it's highly unlikely it will ever change though